### ISSUE TITLE
[Feature request] backup "DEVICE" contacts only (avoid duplicates)

### ISSUE THREAD

**Issue by passthesh3ll** [vmx/sdcontacts#5]
Hello, could you implement an option to backup "DEVICE" contacts only? At the moment each backup has duplicates because are exported also whatsapp numbers, google numbers etc.. This happens because applications like whatsapp copy all numbers internally in orther to use them in the app. If you can implement a function to select which contact list to backup (just like simple contacts app does), it could be very useful in order to backup a clean list without duplicates. Thanks

**Comment by vmx** [comment]
I tried to make the export being the same as if you would export it through the contacts app, when you go to contacts -> settings ->  export. Does it mean that the file exported from this app, is different from the default contacts app? If yes, then I'll have look.

**Comment by passthesh3ll** [comment]
Yes it's different because that app allows you to choose which sources to use for the extraction. If you choose all sources maybe the output will be the same as the contact app. Anyway I didn't use the contact app for this reason, I need a clean backup. (You can find simple contact in fdroid, it's foss)
